samples/iolib/src/main/cpp/player/SampleSource.h

```diff
@@ -46,13 +46,13 @@ class SampleSource: public DataSource {
 
     void setPlayMode(bool fromPause = true) {
         mIsPlaying = true;
-        if (fromPause) {
+        if (fromPause == true) {
```

Comment:
Q: Why original `if(fromPause)` doesn't work?

Reply:
Addressed in the latest commit